bot.js

```diff
@@ -156,7 +156,8 @@ Tx: <${etherscanlink}>
 Tx: ${etherscanlink}
 Url: ${task.gitHubIssue ? displayIssueTwitter(task.gitHubIssue) : 'not found'}
 `
-  
+  LINK_CLAIM = 'https://mark.provable.dev/?ipfs=QmdzB48faGpTVZLjPAZA6jLjhZGbdSmX9vN3vcaf5n7MBy?issue='
+  msg_discord = task.gitHubIssue ? msg_discord + `Claim Url: ${LINK_CLAIM}${task.gitHubIssue.split('issues/')[1]}` : msg_discord
```

Comment:
See https://api.github.com/repos/the-laurel/laurels/issues/153 - I suggest `task.gitHubIssue.number`.